src/Components/OATPropertyEditor/Utils.ts

```diff
@@ -261,4 +262,13 @@ export const handleIdChange = (
     } else {
         setIdErrorLength(true);
     }
+    if (setIdWarning && value.length > 0) {
+        setIdWarning(true);
+    } else {
+        setIdWarning(false);
```

Comment:
setIdWarning could be null here?

Reply:
Done

Reply:
I mean the function ref could be null, not the value 😄 

We should probably do something like:
```
if (setIdWarning) {
  setIdWarning(value.length > 0);
}
```

Reply:
Ho ok got it, Change is submited